python/metatomic_torch/metatomic/torch/__init__.py

```diff
@@ -49,13 +52,63 @@
 from .systems_to_torch import systems_to_torch  # noqa: F401
 
 
-def __getattr__(name):
-    # lazy import for ase_calculator, making it accessible as
-    # ``metatomic.torch.ase_calculator`` without requiring a separate import from
-    # ``metatomic.torch``, but only importing the code when actually required.
-    if name == "ase_calculator":
-        import metatomic.torch.ase_calculator
+# Define the attributes to be lazily imported.
+# XXX(rg): This is a stripped version of lazy-loader [1,2]
+#          Consider growing a dependency on it
+# [1]: https://scientific-python.org/specs/spec-0001/
+# [2]: https://pypi.org/project/lazy-loader/
+# The key is the name exposed to the user.
+# The value is a tuple: (relative_module_path, attribute_name_in_module)
+# If attribute_name_in_module is None, the entire module is returned.
+_lazy_imports = {
+    # Exposes `metatomic.torch.ase_calculator` as `metatomic.torch.ase`
+    "ase": (".ase_calculator", None),
+    # Exposes `MetatomicCalculator` from the submodule as `MetatomicAseCalculator`
+    "MetatomicAseCalculator": (".ase_calculator", "MetatomicCalculator"),
+}
```

Comment:
I don't think we want both `metatomic.torch.ase_calculator` and `metatomic.torch.ase` (and same for `metatomic.torch.ase_calculator.MetatomicCalculator` and `metatomic.torch.MetatomicAseCalculator`), it is going to be confusing to users and to us when debugging errors.